ui/components/NodeInputButton.tsx

```diff
@@ -11,7 +11,14 @@ export const NodeInputButton: FC<NodeInputProps> = ({
   dispatchSubmit,
 }) => {
   const handleClick = (e: MouseEvent | FormEvent) => {
-    void setValue(attributes.value as string).then(() => dispatchSubmit(e));
+    if (attributes.onclick) {
+      // handle passkeys that bring their own trigger
+      e.stopPropagation();
+      e.preventDefault();
+      eval(attributes.onclick);
```

Comment:
question: isn't there any other way to do this than with `eval`? Can we just invoke cast `attributes.onclick` to the right signature function and invoke it? @edlerd

Reply:
we don't do the eval anymore